### PR TITLE
Retrieve data from local host endpoint

### PR DESCRIPTION
**Pull request by Naomi-Dennis**

### CODE REVIEW

File: src/Main.elm [hunk 1]
```diff
@@ -3,42 +3,14 @@ port module Main exposing (elmToJS)
...
-
-
-decodedData =
-    decodeString arrivalTimeDecoder fakeArrivalData
-
-
-fakeArrivalTimes =
-    validateArrivalData decodedData
+type Response
+    = Something (Result Http.Error ArrivalTimes)
```
Comment (damonkelley) [inline src/Main.elm]: What is `Something`? Maybe we can find a more suitable name?

**Review by damonkelley — COMMENTED** [review]
Looks good to me. Let's merge it once we have a better name for `Something`!

File: src/Main.elm [hunk 2]
```diff
@@ -3,42 +3,14 @@ port module Main exposing (elmToJS)
...
-
-
-decodedData =
-    decodeString arrivalTimeDecoder fakeArrivalData
-
-
-fakeArrivalTimes =
-    validateArrivalData decodedData
+type Response
+    = RecievedData (Result Http.Error ArrivalTimes)
```
Comment (hnlee) [inline src/Main.elm]: Trivial nitpick but should be spelled `ReceivedData`

File: src/Main.elm [hunk 3]
```diff
@@ -69,17 +41,15 @@ arrivalBarGraph : Spec
...
-            dataFromColumns []
-                << dataColumn xAxisName (strs fakeArrivalTimes.day)
-                << dataColumn yAxisName (nums fakeArrivalTimes.meanArrivalDifferences)
+            dataFromUrl "http://localhost:4000/" []
 
         enc =
             encoding
-                << position X [ pName xAxisName, pOrdinal, pSort [ soCustom (strs fakeArrivalTimes.day) ], pAxis [ axTitleFontSize axisFontSize ] ]
-                << position Y [ pName yAxisName, pQuant, pAxis [ axTitleFontSize axisFontSize ] ]
+                << position X [ pName "day", pOrdinal, pSort [], pAxis [ axTitleFontSize axisFontSize ] ]
```
Comment (hnlee) [inline src/Main.elm]: If you're going to set the axis names directly as strings here, then do you still need the `xAxisName` and `yAxisName` variables above?

I do think that it's easier to read if you keep them as variable names and set the values separately.

File: src/Main.elm [hunk 4]
```diff
@@ -69,17 +41,15 @@ arrivalBarGraph : Spec
...
     let
         des =
-            description "A simple bar chart with embedded data."
+            description "A visualization of arrival time averages for the MTA G Train Greenpoint Ave. stop"
 
         data =
-            dataFromColumns []
-                << dataColumn xAxisName (strs fakeArrivalTimes.day)
-                << dataColumn yAxisName (nums fakeArrivalTimes.meanArrivalDifferences)
+            dataFromUrl "http://localhost:4000/" []
```
Comment (hnlee) [inline src/Main.elm:47]: Is it going to be retrieving from `localhost` when deployed? Or should this be an env var?

**Review by hnlee — CHANGES_REQUESTED** [review]

File: src/Main.elm (hunk 4 above)
Comment (Naomi-Dennis) [reply src/Main.elm:47]: Environment variables are a bit tricky in elm. I'll have to do more research and add it in the next iteration

**Pull request merged**